Jahia/npm-solid-react-templateset-example
# Update file structure according to the new specs

Updated the file structure to the new specs (https://jira.jahia.org/browse/BACKLOG-22550)

Note : the new "vendors" folder in the spec is actually not a vendors folder because it contains the client js, so we keep the javascript name

Done : 
   -  Moved locales and javascript to the new public folder
   - Moved all the config files of the module to the configuration folder
   - Improved readMe file
   - Updated static resources in package.json

Tests :

 1) Check that the files structure correspond to the specs
 2) Build and deploy the project to a jahia with the lattest version of npm-module-engine and serverSettings and check that :
    - content-type icons are visible when adding content to a page
    - template preview image is visible when creating a site
    - locales are working
    - Logo and favicon are visible
    - Client JS is working when logging
    - content-editor-forms is working (adding an image to an agency is possible)
